src/modules/multiple-authors/multiple-authors.php

```diff
@@ -1561,6 +1568,11 @@ public function action_sync_post_author()
             }
         }
 
+        public function action_sync_author_slug()
```

Comment:
There is no action hook calling this function. We need something like this to make it works:

```
add_action('multiple_authors_sync_author_slug', [$this, 'action_sync_author_slug']);
```